.github/workflows/sandbox-build-push.yml

```diff
@@ -78,10 +78,19 @@ jobs:
           role-to-assume: arn:aws:iam::538673716275:role/github-actions-role
           aws-region: ap-southeast-2
 
+      - name: Wait for tests to succeed
+        uses: lewagon/wait-on-check-action@v1.3.1
```

Comment:
Is there a simpler way to accomplish this? It seems like an involved piece of code for what fundamentally sounds like a check and sleep loop. ?

Reply:
it is, sorry, no, `on: workflow_run` doesn't wait for all the workflows to finish but is triggered each time one of workflows finishes. The best I can do is like this, triggered by one, and wait until the other one finishes.